taxi_service/settings.py

```diff
@@ -20,12 +22,14 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
-)
+SECRET_KEY =\
+    config(
+        "SECRET_KEY",
+        "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
```

Comment:
Ensure that the `SECRET_KEY` is securely managed and not exposed in the codebase. Using `python-decouple` to load it from environment variables is a good practice.